### ISSUE TITLE
Booleans are passed as Strings to filters

### ISSUE THREAD

**Issue by jonjanisch** [yogthos/Selmer#196]
I just started using Selmer and created a simple filter which converts a boolean to 0 or 1. It was failing because instead of true or false the value "true" or "false" is passed instead.

To test I created a simple filter to print the type passed to the filter to the console.

`(add-filter! :hello (fn [x] (println (.getClass x)) (str "Hello " x)))`

It's inconsistent with all other types:

`(render "{{ a|hello }}", {:a true})` -> java.lang.String
`(render "{{ a|hello }}", {:a 1})` -> java.lang.Long
`(render "{{ a|hello }}", {:a (java.util.Date.)})` -> java.util.Date
`(render "{{ a|hello }}", {:a (Boolean. true)})` -> java.lang.String

**Comment by yogthos** [comment]
I agree it makes sense for boolean types to be preserved, and thanks for the PR with the fix. I just pushed out `1.12.12` with the update.
